### ISSUE TITLE
ctl update conf: Query

### ISSUE THREAD

**Issue by sandeepkalra** [ochinchina/supervisord#167]
I have not tested the system entirely, but can I update the conf file and issue "supervisord ctl update". I see reload. Is this equivalent to "update" of supervisord(of python). I want to reload conf, and only restart the new app (without modifying the running programs)

**Comment by ochinchina** [comment]
@sandeepkalra is the supervisorctl python tool used? or use "supervisord ctl" command?

**Comment by thisforeda** [comment]
@ochinchina 

是的，python 版的 supervisor 有一个 supervisorctl update 命令

就是如果配置没有被更改的话，这条命令无作用

但是比如你要是修改了一个 program 的 command 或者任意字段，这条命令会根据实际配置重启或者
停止 program（autostart 有关）。

如果这个 program 属于一个组，那么这条命令会重启整个组。

这条命令某些情况下确实很有用，我经常会定时从脚本里改配置，省得了整个 reload。

**Comment by thisforeda** [comment]
这在比如运行着 redis，但是临时需要修改一下别的服务配置的时候真的很有用。

**Comment by sandeepkalra** [comment]
I was just comparing to the python version of supervisor program to this project. There is a feature called 'update' (in python version) where you simply change the conf, and 'supervisor ctl update' reloads the conf and starts the new process. What is the equivalent here for golang version ? In code, I see 'reload' . If this is not the same, then can you take this as a feature request with support of using REST call to do so?.

**Comment by ochinchina** [comment]
@sandeepkalra Yes, the reload will reload the configuration file and restart all the program.

**Comment by vamega** [comment]
Is there a way to reload the configuration file but only restart certain programs?

**Comment by sandeepkalra** [comment]
@vamega I think there is not. It is nice to have a feature where we pass reload with argument(s) the list of program-groups to reload.

**Comment by ochinchina** [comment]
please use "supervisord ctl reload" command
